dask_expr/_shuffle.py

```diff
@@ -245,6 +254,22 @@ def from_abstract_shuffle(cls, expr: Shuffle) -> Expr:
                         options,
                     )
 
+            if isinstance(partitioning_index, Expr):
+                if partitioning_index.ndim == 1:
+                    col = "_partitions_1"
```

Comment:
nit: consistency with the enumeration in else
```suggestion
                    col = "_partitions_0"
```